app/views/stations/edit.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Edit Station Info</h1>
+
+<form action='/stations/<%= @station.id %>' method = "POST">
+  <input type="hidden" name="_method" value="PUT" />
+  <label for="">Station Name</label>
+  <input type="text" name="station[name]" value="<%= @station.name %>" />
+  <label for="">City</label>
+  <input type="text" name="city[name]" value="<%= @station.city.name %>"/>
```

Comment:
Recommend using `station[city]` here in place of `city[name]`. This will require the updates I mentioned above, but then will put all of your Station params into one hash.